build-system/tasks/presubmit-checks.js

```diff
@@ -707,6 +707,19 @@ const forbiddenTerms = {
       'src/inabox/amp-inabox.js',
     ],
   },
+  'isRenderBlocking': {
+    message:
+      'This is a protected API. Please only override it the element is ' +
+      'render blocking',
+    whitelist: [
+      'src/service/resources-impl.js',
```

Comment:
Signed... I need to make change to all four files to pass the `isRenderBlocking()` method from a component to the resource manager...